macaroonbakery/bakery.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2016 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+import json
+import webbrowser
+
+import macaroons
+import requests
+
+
+def deserialize(json_macaroon):
+    '''Deserialize a JSON macaroon into a macaroon object.
```

Comment:
Perhaps say what kind of macaroon object it is?
I think we're using libmacaroons here, right?
I'm not sure that the docs make it clear what kind
of macaroons we're using.